gluco-check-frontend/src/pages/EditSettings.tsx

```diff
@@ -1,11 +1,10 @@
-import React, { useContext } from "react";
+import { useContext } from "react";
```

Comment:
Yup, eslint is now even indicating it's an 'unused import' :D
Btw... Don't merge yet. I messed up something in the deployment workflow: frontend isn't picking up the env variables

Reply:
👍won't merge thanks for the heads up! If you can, please force-push your lwc, I rebased on main and force-pushed (sorry, thought it would be harmless!) and now am seeing an apparent false positive coverage miss in codecov that I don't think was there before, so if we can just go back to the previous history maybe that'll save some time spent chasing that down?

Reply:
Force-push done! ✅  
Coverage false positive remains, but I'll ignore that. Coverage hasn't changed, but total number of lines has so it's reporting a 0.01% decrease.